projects/gnoland/gno.land/p/eve000/event/agenda/agenda.gno

```diff
@@ -15,16 +15,16 @@ type Agenda struct {
 	Location       *location.Location
 	StartDate      time.Time
 	EndDate        time.Time
-	KeynoteHeading *component.Content
+	Banner *component.Content
```

Comment:
Renamed this multi-use field - it's a convenient place to change status messages or updates about the event